src/main/java/com/kodilla/ecommercee/domain/Cart.java

```diff
@@ -21,10 +21,26 @@
     @Column(name = "CART_ID")
     private long id;
 
-    @OneToOne
+    @OneToOne(cascade = CascadeType.ALL)
     @JoinColumn(name = "USER_ID")
     private User user;
 
+    @ManyToMany(cascade=CascadeType.ALL, mappedBy = "carts")
```

Comment:
Ustalaliśmy, że będziemy korzystać w encjach z lomboka i mapowania dodawać do pól a nie getterów i setterów, dopasuj to prosze do tego.
Poza tym znowu nadużycie cascadeType. Operacje na koszyku nie powinny być propagowane do produktu.